docs/modules/user-guide/partials/con_using-a-top-level-attribute.adoc

```diff
@@ -0,0 +1,38 @@
+[id="con_using-top-level-attribute_{context}"]
+= Using a top-level attribute
+
+[role="_abstract"]
+
+Use top-level attributes in devfiles to reuse global attributes in different sections of the same devfile.
+
+.Devfile variable
```

Comment:
.Procedure

Add a devfile variable so you can add a top-level attribute. 

< Procedure Steps > (see my comment below for these steps)